### DISCUSSION TITLE
Spring Modulith 0.2 released

### DISCUSSION THREAD

**Discussion by odrotbohm** [spring-projects/spring-modulith#93] [Announcements]
## :star: New Features

- Expose application module structure via actuator endpoint [#87](https://github.com/spring-projects-experimental/spring-modulith/issues/87)
- Postgres event publication registry schema should follow Postgres best practices [#81](https://github.com/spring-projects-experimental/spring-modulith/issues/81)
- Improve module integration by simplifying event listener declaration [#80](https://github.com/spring-projects-experimental/spring-modulith/issues/80)

## :hammer: Dependency Upgrades

- Upgrade to Spring Boot 3.0.1 [#88](https://github.com/spring-projects-experimental/spring-modulith/issues/88)

**Comment by Rohay-Lee** [comment]
您好，邮件已收到，谢谢！李罗海
